src/services/create-default-svc-service.ts

```diff
@@ -0,0 +1,150 @@
+import * as fs from 'node:fs'
```

Comment:
Not sure if a separate service class is needed here. Couldn't this be just merged into `svc-service`?

Reply:
I may agree in splitting them as they have different responsibilities and class could be difficult to read